source/extensions/omni.isaac.lab/omni/isaac/lab/scene/interactive_scene_cfg.py

```diff
@@ -85,4 +85,8 @@ class MySceneCfg(InteractiveSceneCfg):
     """
 
     replicate_physics: bool = True
-    """Enable/disable replication of physics schemas when using the Cloner APIs. Default is True."""
+    """Enable/disable replication of physics schemas when using the Cloner APIs. Default is True.
+
+    If true, the environment will be assumed to be homogeneous and cloning happens on a global scope.
+    Set this to False for heterogeneous environments, where cloning happens per-object.
```

Comment:
Homogenous and hetergoneous cloning might be less intuitive?

Also we should document this in the Scene class docstrings as well. Also it seems that the documentation on usage there is limited. Can we fix that too?

```suggestion
    If true, the simulation will have the same asset instances (USD prims) in all the cloned environments. Internally, this ensures optimization
    in setting up the scene and parsing it via the physics stage parser.
    
    If False, the simulation allows having separate asset instances (USD prims) in each environment. This flexibility comes
    at a cost of slowdowns in setting up and parsing the scene.
    
    .. note::
        Optimized parsing of certain prim types (such as deformable objects) is not currently supported
        by the physics engine. In these cases, this flag needs to be set to False.
```